### DISCUSSION TITLE
Does not show events that long more than a day

### DISCUSSION THREAD

**Discussion by dimabin** [nhn/tui.calendar#1350] [Q&A] [answered]
If I create an event that longs more than a day (for example, if it starts on tuesday 10:00 and ends on wednesday 12:00), it is not shown in the calendar.

## Current Behavior

```js
// Write example code
```

**Comment by dimabin** [comment]
Sorry, it does show it - but in the "All day" section. Is there a way to show such events in the main section?

**Comment by adhrinae** [comment, marked as answer]
Unfortunately, It's a known limitation of the model calendar events.

For more info, check out [the documentation](https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/event-object.md#isallday).
